docs/commands/New-PesterConfiguration.mdx

```diff
@@ -0,0 +1,240 @@
+---
+id: New-PesterConfiguration
+title: New-PesterConfiguration
+description: Help page for the Powershell Pester "New-PesterConfiguration" command
+keywords:
+  - Powershell
+  - Pester
+  - Help
+  - Documentation
+hide_title: false
+hide_table_of_contents: false
+custom_edit_url: null
+---
+
+## SYNOPSIS
+
+Creates a new [PesterConfiguration] object for advanced configuration of Invoke-Pester.
+
+## SYNTAX
+
+```powershell
+New-PesterConfiguration [[-Hashtable] <IDictionary>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The New-PesterConfiguration function creates a new [PesterConfiguration] object
+to enable advanced configurations for runnings tests using Invoke-Pester.
+
+Without parameters, the function generates a configuration-object with default
+options.
+The returned [PesterConfiguration] object can be modified to suit your
+requirements.
+
+Calling New-PesterConfiguration is equivalent to calling [PesterConfiguration]::Default which was used in early versions of Pester 5.
+
+Sections and options:
+
+Run:
+  Path: Directories to be searched for tests, paths directly to test files, or combination of both.
+  Default value: @('.')
+
+  ExcludePath: Directories or files to be excluded from the run.
+  Default value: @()
+
+  ScriptBlock: ScriptBlocks containing tests to be executed.
+  Default value: @()
+
+  Container: ContainerInfo objects containing tests to be executed.
+  Default value: @()
+
+  TestExtension: Filter used to identify test files.
+  Default value: '.Tests.ps1'
+
+  Exit: Exit with non-zero exit code when the test run fails.
+When used together with Throw, throwing an exception is preferred.
+  Default value: $false
+
+  Throw: Throw an exception when test run fails.
+When used together with Exit, throwing an exception is preferred.
+  Default value: $false
+
+  PassThru: Return result object to the pipeline after finishing the test run.
+  Default value: $false
+
+  SkipRun: Runs the discovery phase but skips run.
+Use it with PassThru to get object populated with all tests.
+  Default value: $false
+
+Filter:
+  Tag: Tags of Describe, Context or It to be run.
+  Default value: @()
+
+  ExcludeTag: Tags of Describe, Context or It to be excluded from the run.
+  Default value: @()
+
+  Line: Filter by file and scriptblock start line, useful to run parsed tests programatically to avoid problems with expanded names.
+Example: 'C:\tests\file1.Tests.ps1:37'
+  Default value: @()
```

Comment:
```suggestion
  Line: Filter by file and scriptblock start line, useful to run parsed tests programatically to avoid problems with expanded names. Example: 'C:\tests\file1.Tests.ps1:37'
  Default value: @()
```